package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.7.0",
+    "@types/react-redux": "^7.1.16",
```

Comment:
Any `@types` should be added in the `devDepenedencies` not the actual dependencies because `@types` is for type definitions only

Reply:
Corrected